replication.go

```diff
@@ -330,14 +330,13 @@ func NewContentManager(db *gorm.DB, api api.Gateway, fc *filclient.FilClient, tb
 		Replication:                  cfg.Replication,
 		tracer:                       otel.Tracer("replicator"),
 		DisableFilecoinStorage:       cfg.DisableFilecoinStorage,
-		IncomingRPCMessages:          make(chan *drpc.Message),
+		IncomingRPCMessages:          make(chan *drpc.Message, 200000),
```

Comment:
Any reason why it's fixed as `200000`? Can we parameterize this and use `200000` by default.

Reply:
no reason actually, I will move it to be configurable, that way we can play around a variable size